docs/documentation/configuration/index.mdx

```diff
@@ -85,3 +82,21 @@ longer-term search overhead of having many additional index segments until (auto
 ```sql
 SET paradedb.statement_memory_budget = 15;
 ```
+
+## Index Configuration Settings
+
+These settings can be applied to an existing `USING bm25` index with `ALTER INDEX <idxname> SET (...)` and affect runtime
+operations around how the underlying tantivy index segments are merged, and when.
+
+`target_segment_count` is an integer value that indicates the number of tantivy index segments that are preferred for
+the index. If unspecified, the default is the number of available CPUs. The actual number of index segments may be
+different, depending on runtime operations, but should generally be close to this value.
+
+`merge_on_insert` is a boolean that indicates if pg_search should merge segments whenever a tuple is inserted into the index.
+This includes INSERT/UPDATE/COPY statements. The default is `true`. Setting this to `false` can improve INSERT/UPDATE/COPY
+throughput at the expense of creating more segments that will be later merged by (auto)VACUUM.
+
+```sql
+ALTER INDEX idxfoo_bm25_index SET (target_segment_count = 8);
```

Comment:
What would be the easiest way to make it clear to the user that their index name has `_bm25_index` appended to it?

Reply:
To stop doing that!

But until we get rid of the create_bm25 function, I don’t really have any good suggestions. I kinda figure if you’re a user that’s gonna change these settings then they’ll know the index name?

Reply:
Issue is we append `bm25_index` to the index name without telling people that it happens. We can totally stop doing that now, btw. I'll approve for now.

Reply:
Yeah, I understand.  I'm just saying if the user is motivated enough to change these particular settings then they'll likely know what their index is named.

If we can stop doing that now, that'd be great and we should PR that.  However, I think our time is better spent getting rid of `create_bm25()`.